src/Adapter/Order/CommandHandler/AddOrderPaymentHandler.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Order\CommandHandler;
+
+use Currency;
+use OrderInvoice;
+use PrestaShop\PrestaShop\Adapter\Order\AbstractOrderHandler;
+use PrestaShop\PrestaShop\Core\Domain\Order\Command\AddOrderPaymentCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\CommandHandler\AddOrderPaymentHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Order\Exception\OrderException;
+use Validate;
+
+/**
+ * @internal
+ */
+final class AddOrderPaymentHandler extends AbstractOrderHandler implements AddOrderPaymentHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(AddOrderPaymentCommand $command)
+    {
+        $order = $this->getOrderObject($command->getOrderId());
+
+        $amount = str_replace(',', '.', $command->getPaymentAmount());
```

Comment:
😭

Reply:
Should I change it? I'm a bit worried about changing legacy code, even this one. :/

Reply:
No, we cannot change it, at least without an heavy test coverage for this usecase.
When we have working tests for this, we can then modify it and check nothing is broken.